src/bundle/Resources/public/js/scripts/core/draggable.js

```diff
@@ -164,7 +175,10 @@
             this.itemsContainer.addEventListener('drop', this.onDrop, false);
 
             doc.body.addEventListener('dragover', (event) => {
-                if (!this.itemsContainer.contains(event.target)) {
+                if (
```

Comment:
This fragment is responsible for removing placeholder after mouse leaves drag container, but earlier it was also removing placeholder while dragging over placeholder...